onnxruntime/python/tools/quantization/README.md

```diff
@@ -101,4 +101,26 @@ If False, the inputs/activations are quantized using dynamic scale and zero poin
             {
                 'resnet_model/Relu_3:0': [np.int8(0), np.float32(0.011359662748873234)],
                 'resnet_model/Relu_4:0': [np.uint8(0), np.float32(0.011359662748873234)]
-            }
\ No newline at end of file
+            }
+
+## Calibration
+
+Calibration can be used to improve quantization techniques, adding reduced-precision computation for neural networks while retaining high accuracy without retraining. `calibrate.py` adds ReduceMin and ReduceMax nodes to all Conv and MatMul nodes in a loaded ONNX model and ensures their outputs are stored as part of the graph output, extracts intermediate output values after inference, and returns a dictionary mapping added node names to average (ReduceMin, ReduceMax) values as input to `quantize.py`. Example usage:
+```python
+import onnx
+import calibrate
+
+# Generating augmented ONNX model
+model_path = input('Full filepath to model: ')
+model = onnx.load(model_path)
+augmented_model = augment_graph(model)
```

Comment:
all this should be part of the calibrate script... why bother user with augmented model?
I think we discussed this yesterday - add all this to the calibrate script and only have the user pass in model paths, path to calibration data set and so

Reply:
Example Usage should show how to call the script
Something like Calibrate.py --model_path <> etc...

Reply:
Done!